spec/services/preserved_object_handler_confirm_version_spec.rb

```diff
@@ -295,23 +283,21 @@
         expect(po).not_to have_received(:save!)
         expect(pc).to have_received(:save!)
       end
-      it 'calls PreservedCopy.touch (but not PreservedObject.touch) if the existing record is NOT altered' do
+      it 'calls PreservedCopy.save! if the existing record is NOT altered' do
```

Comment:
save as in check_existence spec:  this test shows that PreservedCopy timestamps were updated for the check, but PreservedObject remained unchanged.